lib/template/config/puma.rb

```diff
@@ -1,6 +1,6 @@
 require "./config/config"
 
-environment Config.rack_env
+environment Config.pliny_env
```

Comment:
Dang.

Unfortunately the originally linked blog post now seems to be gone. At the time I remember understanding what they were saying in principle, but not ever having personally experienced any actual problems from just using `rack_env`.

One possible easy fix: just change the defaults for `pliny_env` and `rack_env` to `production`. Alternatively, make them mandatory configuration so that users have to put something in the env.